### DISCUSSION TITLE
Can SFTPgo be configured to run on other ports as 80 and 443 has been taken

### DISCUSSION THREAD

**Discussion by abubin** [drakkan/sftpgo#1796] [Q&A]
Just installed SFTPgo on a Windows Server and would like to configure SSL for webui access. I would like to configure the webui to use other ports for eg 8443. This is because in the Windows server that I installed SFTPgo, IIS is already taken up port 80 and 443.

I read in the documentation that HTTP-01 need to use port 80. And I need to use reverse proxy if I want to use other ports. Are there any guide to running reverse proxy on a Windows server? Do I need to use port 80 actually? Cause I am using HTTPS for webui.

**Comment by drakkan** [comment]
Hello,

We don't have a specific guide for IIS, but you should be able to get it working by configuring a web root as explained [here](https://docs.sftpgo.com/latest/tutorials/lets-encrypt-certificate/#obtaining-a-certificate-using-the-acme-protocol-built-into-sftpgo).
The web root must match the root directory served by IIS. I hope this is helpful

**Comment by abubin** [comment]
It works! I just use the ACME configuration in the webui and it works after restarting SFTPgo service. 

Webui maintained at port 8080.

Thanks for making it so easy!

![image](https://github.com/user-attachments/assets/fb2d48e3-f9e2-4659-9f12-27d46ac75685)

**Comment by abubin** [comment]
Thank you. Case closed.
